src/hooks/useGetTimezones.ts

```diff
@@ -0,0 +1,601 @@
+export const useGetTimezones = (): Array<Record<string, string>> => {
```

Comment:
Yeah... this isn't the best approach but it's how it's done in the angular code base.

Reply:
Guess this is fine. The only other options I saw were https://github.com/moment/luxon/issues/353#issuecomment-514203601 and https://github.com/vvo/tzdb/ but idk that we need to bring in another library for this. They don't exactly give the english formatting we want anyways. Although none of these hardcoded values are translated...

Reply:
I could just do ```import i18n from 'i18next``` and then use ```i18n.t()``` on the strings right?

Reply:
I think that works. This seems like a chore for a translator to work through manually though...

Reply:
Another weird test thing. Whenever I added the translation method around each value, the test now fails. It can't find the element, similar to what was happening to the language selection in the test. I wonder if it's taking too long to load or something to that extent.

Reply:
Figured it out, had to mock ```i18next```